packages/utilities/gel-foundations/src/scripts/arabic-pashto.js

```diff
@@ -0,0 +1,144 @@
+const arabicPashtoTypography = {
+  canon: {
+    groupA: {
+      fontSize: '32',
+      lineHeight: '42',
+    },
+    groupB: {
+      fontSize: '36',
+      lineHeight: '46',
+    },
+    groupD: {
+      fontSize: '44',
+      lineHeight: '54',
+    },
+  },
+  trafalgar: {
+    groupA: {
+      fontSize: '24',
+      lineHeight: '34',
+    },
+    groupB: {
+      fontSize: '26',
+      lineHeight: '36',
+    },
+    groupD: {
+      fontSize: '38',
+      lineHeight: '48',
+    },
+  },
+  paragon: {
+    groupA: {
+      fontSize: '',
+      lineHeight: '',
+    },
+    groupB: {
+      fontSize: '',
+      lineHeight: '',
+    },
+    groupD: {
+      fontSize: '',
+      lineHeight: '',
+    },
+  },
+  doublePica: {
+    groupA: {
+      fontSize: '',
+      lineHeight: '',
+    },
+    groupB: {
+      fontSize: '',
+      lineHeight: '',
+    },
+    groupD: {
+      fontSize: '',
+      lineHeight: '',
+    },
+  },
+  greatPrimer: {
+    groupA: {
+      fontSize: '',
+      lineHeight: '',
+    },
+    groupB: {
+      fontSize: '',
+      lineHeight: '',
+    },
+    groupD: {
+      fontSize: '',
+      lineHeight: '',
+    },
+  },
+  bodyCopy: {
+    groupA: {
+      fontSize: '18',
+      lineHeight: '28',
+    },
+    groupB: {
+      fontSize: '20',
+      lineHeight: '30',
+    },
+    groupD: {
+      fontSize: '22',
+      lineHeight: '34',
+    },
+  },
+  pica: {
+    groupA: {
+      fontSize: '',
```

Comment:
Is there a reason we are leaving these values empty?

Reply:
Because UX hasn't tested some of GEL Types for some scripts, but I populate them with the current values from PAL for now.